src/appengine/handlers/upload_testcase.py

```diff
@@ -405,6 +408,12 @@ def do_post(self):
         raise helpers.EarlyExitException(
             'You are not privileged to set testcase metadata.', 400)
 
+    # If we have a AFL or libFuzzer target, use that for arguments.
+    # Launch command looks like
+    # python launcher.py {testcase_path}
```

Comment:
nit: comment is a little outdated :) Yes for AFL because that hasn't been refactored. Maybe add a TODO to remove this comment and code once everything is migrated since %TESTCASE% isn't needed anymore for new engine impl either.